plugins/outputs/opensearch/opensearch.go

```diff
@@ -23,7 +22,7 @@ import (
 	"github.com/influxdata/telegraf"
 	"github.com/influxdata/telegraf/config"
 	"github.com/influxdata/telegraf/internal/choice"
-	httpconfig "github.com/influxdata/telegraf/plugins/common/http"
+	commontls "github.com/influxdata/telegraf/plugins/common/tls"
```

Comment:
There was a conflict previously. Do you insist on naming it `tls`? Otherwise I would stick with the current naming to avoid potential future conflicts...

Reply:
I don't insist. It was just a remark as it is done like that in many other plugins.

They usually name it `tlsint`:
```
  79    "github.com/influxdata/telegraf/plugins/common/tls"
   1    _tls "github.com/influxdata/telegraf/plugins/common/tls"
   4    commontls "github.com/influxdata/telegraf/plugins/common/tls"
   1    influxtls "github.com/influxdata/telegraf/plugins/common/tls"
   4    internaltls "github.com/influxdata/telegraf/plugins/common/tls"
   1    itls "github.com/influxdata/telegraf/plugins/common/tls"
   1    tlsConfig "github.com/influxdata/telegraf/plugins/common/tls"
  20    tlsint "github.com/influxdata/telegraf/plugins/common/tls"
```